samples/bot-sharepoint-file-viewer/csharp/README.md

```diff
@@ -0,0 +1,102 @@
+﻿﻿﻿Using this C# sample, a bot can upload file to SharePoint site and same files can be viewed in Teams file viewer.
```

Comment:
a bot with capability to upload files to SharePoint site and same files can be viewed in Teams file viewer